lib/rules/indent.js

```diff
@@ -119,7 +119,10 @@ module.exports = function(context) {
     function checkNodesIndent(nodes, indent, excludeCommas) {
         nodes.forEach(function(node) {
             var nodeIndent = getNodeIndent(node, false, excludeCommas);
-            if (nodeIndent !== indent && isNodeFirstInLine(node)) {
+            if (
```

Comment:
I'm not really sure. I can probably filter `nodes` in `checkIndentInArrayOrObjectBlock` before passing here but the current implementation feels better since `ArrayExpression` and `ObjectExpression` nodes will be covered autoamtically (since they are defined in exports).

Does this make sense?